internal/commands/app/create.go

```diff
@@ -65,7 +101,27 @@ func (cmd *CommandCreate) Handler(profile *cli.Profile, ui terminal.UI, clients
 		return err
 	}
 
-	// TODO(REALMC-8134): Implement dry-run for app create command
+	var ds dataSource
+	if cmd.inputs.DryRun {
+		if cmd.inputs.DataSource != "" {
+			ds, err = cmd.inputs.resolveDataSource(clients.Atlas, groupID)
```

Comment:
so it seems like `resolveDataSource` is invoked regardless of whether we're in a dry run or not.  do you think it'd be worth doing that before this `DryRun` check?

```go
var ds dataSource
if cmd.inputs.DataSource != "" {
  ds, err = cmd.inputs.resolveDataSource(clients.Atlas, groupID)
  if err != nil {
    return err
  }
}

if cmd.inputs.DryRun {
  // printing logic here
  return nil
}

// further down in the handler
if cmd.inputs.DataSource != "" {
  var dataSourceDir string
  switch loadedApp.ConfigVersion() { ... }
}

// run import...
```